docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -17,16 +17,14 @@ a DAML ledger on top of their distributed-ledger or database of choice.
 Use this tool to verify if your Ledger API endpoint conforms to the :doc:`DA
 Ledger Model </concepts/ledger-model/index>`.
 
-Please note that currently the tool is in experimental status.
```

Comment:
You can remove this, but the tool is still bound to change. Probably better not to.

Reply:
yeah i think we are beyond experimental though : more like alpha/beta state i'd say